src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,245 @@
+package duke;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import duke.exception.*;
+import duke.task.*;
+import java.io.*;
```

Comment:
It would be good to explicitly state what classes would be used and maintained, like in the [import statement convention](https://se-education.org/guides/conventions/java/basic.html#package-and-import-statements).